src/NITTA/Model/Networks/Bus.hs

```diff
@@ -733,3 +777,77 @@ instance (UnitTag tag, VarValTime v x t) => Testable (BusNetwork tag v x t) v x
 isDrowAllowSignal Sync = bool2verilog False
 isDrowAllowSignal ASync = bool2verilog True
 isDrowAllowSignal OnBoard = "is_drop_allow"
+
+--------------------------------------------------------------------------
+
+data BuilderSt tag v x t = BuilderSt
```

Comment:
Do we actually need such builder? May be can do that by your new API with few helper functions?

Reply:
When you build a network from configuration files, the bnRemains will be empty, so there will be no options for prototype allocation either.